scripts/migrations/migration_NEXT_userData.js

```diff
@@ -0,0 +1,50 @@
+const fs = require("fs");
+const knex = require("knex");
+const path = require("path");
+const yargs = require("yargs");
+
+const migrateUserDataView = async (configDirectory, writeMode) => {
+    const configPath = path.resolve(configDirectory, "mainConfig.json");
+    const configJSON = JSON.parse(fs.readFileSync(configPath, { encoding: "utf-8" }));
+
+    const userDataSchema = path.resolve("scripts", "sql", "003-user-data.sql");
+    if (fs.existsSync(userDataSchema)) {
+        const connection = await knex({ client: "pg", connection: configJSON.database });
+        if (await connection.schema.hasColumn("user_data_list", "data_content")) {
```

Comment:
Il manque pas un not ici ?
 if ( ! await connection.schema.hasColumn("user_data_list", "data_content")) {